### PR TITLE
Machine Descriptions

### PR DESCRIPTION
**Pull request by Geevies**
* Gave various machines descriptions.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/game/machinery/computer/prisoner.dm [hunk 1]
```diff
@@ -1,8 +1,6 @@
-//This file was auto-corrected by findeclaration.exe on 25.5.2012 20:42:31
-
 /obj/machinery/computer/prisoner
 	name = "prisoner management console"
-
+	desc = "A console that can be used to track tracking implants, and deliver the activation signal to chemical implants."
```
Comment (alsoandanswer) [inline code/game/machinery/computer/prisoner.dm:3]:
```suggestion
	desc = "A console that can be used to track tracking implants, and deliver an activation signal to chemical implants."
```

File: code/game/machinery/kitchen/cooking_machines/candy.dm [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /obj/machinery/appliance/mixer/candy
 	name = "candy machine"
-	desc = "Get yer candied cheese wheels here!"
```
Comment (alsoandanswer) [inline code/game/machinery/kitchen/cooking_machines/candy.dm:3]: I suggest keeping this as a fluff desc.

File: code/game/machinery/kitchen/cooking_machines/cereal.dm [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /obj/machinery/appliance/mixer/cereal
 	name = "cereal maker"
-	desc = "Now with Dann O's available!"
```
Comment (alsoandanswer) [inline code/game/machinery/kitchen/cooking_machines/cereal.dm:3]: I suggest keeping this as a fluff desc.

File: code/game/machinery/kitchen/cooking_machines/fryer.dm [hunk 4]
```diff
@@ -1,6 +1,6 @@
 /obj/machinery/appliance/cooker/fryer
 	name = "deep fryer"
-	desc = "Deep fried <i>everything</i>."
```
Comment (alsoandanswer) [inline code/game/machinery/kitchen/cooking_machines/fryer.dm:3]: I suggest keeping this as a fluff desc.

File: code/game/machinery/kitchen/cooking_machines/oven.dm [hunk 5]
```diff
@@ -1,6 +1,6 @@
 /obj/machinery/appliance/cooker/oven
 	name = "oven"
-	desc = "Cookies are ready, dear."
```
Comment (alsoandanswer) [inline code/game/machinery/kitchen/cooking_machines/oven.dm:3]: I suggest keeping this as a fluff desc.

File: code/game/machinery/kitchen/cooking_machines/candy.dm (hunk 2 above)
Comment (Geevies) [reply code/game/machinery/kitchen/cooking_machines/candy.dm:3]: nah i don't think that's worth a desc_fluff

**Pull request merged**